transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+In this configuration, the server will provide a certificate to the client (`certificate`) containing the server host name.
+The client will compare the server hostname to the one it knows (`serverName`) and will then validate the provided certificate against its own root certificate (`caCertificate`) confirming the server identity.
+
+:::note
+Root certificate is also sometimes referred as CA Certificate.
+:::
+
+Once its identity confirmed, the server will then use the private key (`privateKey`) to establish the secure communication.
+
+### Requirements
+To use the client server secure workflow, you need a `valid certificate` and the `root certificate` it has been generated from. You also need the `private key` that has been used to create the certificate.
+If you don't have these, they can be generated using OpenSSL. The procedure is detailed hereafter. 
+
+
+
+### Generating the required keys and certificates with OpenSSL
+
+It is assumed that you have [OpenSSL](https://www.openssl.org/) installed on your machine.
+
+#### Generate the Certification Authority root certificate. 
+First thing first is to generate a private key. We will use it later on to generate the Certification Authority root certificate. 
+```shell
+openssl genrsa -out clientPrivateKeyForRootCA.pem 2048
+```
+Now that you have a private key, you can now generate the root certificate.
+
+```shell
+openssl req -x509 -new -nodes -key clientPrivateKeyForRootCA.pem -sha256 -days 1095 -out myGameClientCA.pem
+```
+You will be prompted to answer several questions. Most of the answers are not that important within the present context. 
+It is however useful to use a `common name` that makes sense for you to identify this certificate amongst others.
+Ideally, you would want to use your domain name if you have one.
+
+
+#### Generate the CA-signed certificate to use with the server
+Create now a private key for the server. 
+```shell
+openssl genrsa -out myGameServerPrivate.pem 2048
+```
+From this private key, you can generate a certificate signing request. 
+```shell
+openssl req -new -key myGameServerPrivate.pem -out myGameServerCertificateSigningRequest.pem
+```
+You'll be prompted with the same questions as for generating the root certificate.
+The answers are no more important, except for the `common name` : use the host name of your server if you have one. This common name 
+will appear in the generated certificate and will be compare by the Client against the common name it received as parameter (`serverName`).
+
+Finally, using the different files generated, we can create the certificate file the server will use to authenticate itself : 
+```shell
+openssl.exe x509 -req -in myGameServerCertificateSigningRequest.pem -CA myGameClientCA.pem -CAkey clientPrivateKeyForRootCA.pem -CAcreateserial -out myGameServerCertificate.pem -days 365 -sha256
+```
+You should have now generated a total of five files. Out of these, only will be used later on : 
```

Comment:
I think the last sentence is missing a "3" as in "Out of these, only 3 will be used..."

Reply:
good catch !